tidysic/main.py

```diff
@@ -36,7 +37,7 @@ def dump_config(ctx: click.Context, param: click.Parameter, value: Any) -> None:
 )
 @click.argument("source", type=click.Path(exists=True, file_okay=False))
 @click.argument("target", type=click.Path(exists=False, file_okay=False))
-def run(verbose: bool, config_path: str, source: str, target: str) -> None:
+def run(verbose: bool, config_path: Optional[Path], source: str, target: str) -> None:
```

Comment:
Should `source` and `target` also be typed as `Path` ? Also, they are described above as being of type `click,Path`, which I assume might be different from `pathlib.Path`. I haven't done any reading, but is this correct ?

Reply:
Fixed